kernel/threads-asm.s

```diff
@@ -1,62 +1,33 @@
-.macro push_callee_save_reg
-	push %rbx
-	push %rbp
-	push %r12
-	push %r13
-	push %r14
-	push %r15
-.endm
-
-.macro pop_callee_save_reg
+/* This must stay in sync with 'push_general_purpose_reg' in isr-asm.s, as well
```

Comment:
Is there a reason you can't put these macros in one file and use a `.include`?

Reply:
Merged threads-asm.s and isr-asm.s  RIP threads-asm.s, no one ever really understood you.

Reply:
F